bokehjs/test/unit/models/tools/edit/box_edit_tool.ts

```diff
@@ -87,7 +87,7 @@ describe("BoxEditTool", () => {
       const testcase = await make_testcase()
       const hit_test_stub = sinon.stub(testcase.glyph_view, "hit_test")
 
-      hit_test_stub.returns(create_hit_test_result_from_hits([[1, 0]]))
+      hit_test_stub.returns(Selection.from_hits([[1, 0]]))
```

Comment:
@philippjfr, what was the reason to write code like this? This is equivalent to `new Selection({indices: [1]})`, because what `*from_hits` does is it sorts by the other argument (distance). I suppose this was simply copied from elsewhere?

Reply:
Yes that seems likely. Definitely don't think there were any deep reasons behind it.